### PR TITLE
Use ubuntu for extra GHA workflows

### PR DESCRIPTION
**Pull request by schloerke**
Related: https://github.com/rstudio/bslib/pull/206/files#diff-9c940e8ad2b7bc4c26ec3da57b94bc00e73e2166cfed689da51a4c59bcc0a310R132
Related: https://github.com/rstudio/bslib/pull/206/files#diff-9c940e8ad2b7bc4c26ec3da57b94bc00e73e2166cfed689da51a4c59bcc0a310R138

### CODE REVIEW

File: .github/workflows/pkgdown.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {os: macOS-latest,   r: 'release'}
+          - {os: ubuntu-20.04,   r: 'release', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
```
Comment (cpsievert) [inline .github/workflows/pkgdown.yaml:18]: Part of me wants to keep mac for the aqua capabilities but it's not that important for bslib (it is for thematic, but I suppose we could always fully embrace ragg)

File: .github/workflows/R-CMD-check.yaml [hunk 2]
```diff
@@ -183,16 +187,20 @@ jobs:
...
-              # install master version for deployment
-              "rstudio/bslib",
-              # app deps
-              "BH", "rprojroot", "rsconnect", "ggplot2", "lattice", "rlang", "DT", "reactable"
-            ),
-            upgrade = TRUE
-          )
+          # install package from github for rsconnect
+          pak::pkg_install("renv", upgrade = TRUE)
+          # pak::pkg_install("rstudio/sass", dependencies = TRUE)
```
Comment (cpsievert) [inline .github/workflows/R-CMD-check.yaml:192]:
```suggestion
```

File: .github/workflows/R-CMD-check.yaml [hunk 3]
```diff
@@ -171,6 +172,9 @@ jobs:
         shell: Rscript {0}
         run: |
           pak::local_system_requirements(execute = TRUE)
+          pak::pkg_system_requirements("renv", execute = TRUE)
+          pak::pkg_system_requirements("rsconnect", execute = TRUE)
```
Comment (cpsievert) [inline .github/workflows/R-CMD-check.yaml:176]: Just curious, it this what gives us the sys libs for deployment?

File: .github/workflows/R-CMD-check.yaml [hunk 4]
```diff
@@ -171,6 +172,9 @@ jobs:
...
+          pak::pkg_system_requirements("pkgdown", execute = TRUE)
```
Comment (cpsievert) [inline .github/workflows/R-CMD-check.yaml:177]:
```suggestion
```

File: .github/workflows/R-CMD-check.yaml [hunk 5]
```diff
@@ -171,6 +172,9 @@ jobs:
...
       # xquartz and cairo are needed for Cairo package.
       # harfbuzz and fribidi are needed for textshaping package.
```
Comment (cpsievert) [inline .github/workflows/R-CMD-check.yaml:179]: Let's comment out this block if no longer relevant

File: .github/workflows/pkgdown.yaml (hunk 1 above)
Comment (cpsievert) [reply .github/workflows/pkgdown.yaml:18]: (To put it another way, I'm fine with keeping ubuntu here)

**Review by cpsievert — APPROVED** [review]
LGTM pending minor nitpicks

File: .github/workflows/pkgdown.yaml (hunk 1 above)
Comment (schloerke) [reply .github/workflows/pkgdown.yaml:18]: sounds good

File: .github/workflows/R-CMD-check.yaml (hunk 3 above)
Comment (schloerke) [reply .github/workflows/R-CMD-check.yaml:174]: yup. need to make sure curl and friends are installed. there's a lot of overlap and it _may not_ be absolutely necessary, but it's safe to do

File: .github/workflows/R-CMD-check.yaml (hunk 5 above)
Comment (schloerke) [reply .github/workflows/R-CMD-check.yaml]: renv and rsconnect are needed. I've removed pkgdown

**Comment by schloerke** [comment]
squash merging!

**Pull request merged**